### PR TITLE
Add details about building a distribution.

### PR DESCRIPTION
**Pull request by adamretter**

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 45m 46s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **5036**, Failures: **0**, Skipped: **7** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/pdi/job/PR-7841/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by adamretter** [comment]
@ddiroma @rmansoor @ssamora Hey, I am not sure how to reach the community maintainers. This PR has been open for a few months now... is there something I need to do which I have missed?

File: README.md [hunk 1]
```diff
@@ -46,6 +46,17 @@ you shouldn't as you know)
 
 The build result will be a Pentaho package located in ```target```.
 
+#### Packaging / Distributing it
+
+Packages can be build by using the following command:
```
Comment (ddiroma) [inline README.md:51]: please change to "built" instead of "build"

Comment (adamretter) [reply README.md]: @ddiroma Thanks :-) I have fixed that now

**Comment by adamretter** [comment]
@ddiroma I think the `CodeQL / Analyze (java) (pull_request)` check is unrelated to my changes,

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 47m 25s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **5090**, Failures: **0**, Skipped: **7** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/pdi/job/PR-7841/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by adamretter** [comment]
@ddiroma Are you happy with my additional changes now?

**Comment by adamretter** [comment]
Thank you @ddiroma  :-)

**Pull request merged**